front/src/modules/simulationResult/SimulationResultExport/exportTrainCSV.ts

```diff
@@ -126,10 +118,35 @@ const overloadSteps = (
     time: trainRegime.times[index],
   }));
 
+  const formattedElectricalProfiles = electricalProfiles.values.map((value, index) => {
+    const currentTrainRegime = {
+      speed: interpolateValue(
+        trainRegime,
+        mmToM(electricalProfiles.boundaries[index - 1]) || 0,
+        'speeds'
+      ),
+      position: mmToM(electricalProfiles.boundaries[index - 1]) || 0,
+      time:
+        interpolateValue(trainRegime, mmToM(electricalProfiles.boundaries[index - 1]), 'times') ||
+        0,
+    };
+    if (value.electrical_profile_type === 'profile') {
+      return {
+        electrificationProfile: value.profile || '',
+        ...currentTrainRegime,
+      };
+    }
+    return {
+      electrificationProfile: '',
+      ...currentTrainRegime,
+    };
```

Comment:
To simplify you can minify like this 

```suggestion
   return {
      electrificationProfile:
        value.electrical_profile_type === 'profile' ? value.profile || '' : '',
      ...currentTrainRegime,
}
```